.babelrc

```diff
@@ -3,11 +3,12 @@
     "react"
   ],
   "plugins": [
-    "transform-es2015-modules-amd",
     "check-es2015-constants",
     "transform-es2015-block-scoping",
     "transform-es2015-arrow-functions",
     "transform-es2015-parameters",
+    "transform-es2015-destructuring",
```

Comment:
Uglify harmony branch doesn't support destructuring yet, as well as Edge (this thing doesn't support a ton of stuff, soon I'll try to make it work), so I'd rather transpile this feature as well, what do you think?